src/index.d.ts

```diff
@@ -0,0 +1,52 @@
+import { Transform } from "readable-stream";
+
+export declare module 'encoding' {
```

Comment:
@erikyo this shouldn't be in the types exported by `gettext-parser` it should be added to a local declaration file, and included in the `"compilerOptions"."types"` of the `tsconfig.json` otherwise the declaration of another module is being exported as part of the `gettext-parser` package, which I don't believe is the intent.

Reply:
ah right, but it's really bad to have to make a file just for this dependency which is not typed. Tn the pr #89 where the types were fixed I export it like [in this way](https://github.com/smhg/gettext-parser/blob/42a969325c052b9b1927291b4d6a15172298e21f/src/types.d.ts)... I will try to find a way to avoid it if possible, if you have a proposal please tell me

Reply:
Just checked and seem that the `src/index.d.ts` file is not "copied" into `lib` when building the module... this partially solves the issue you mentioned but, on the other hand i need to find a way to export all the required definitions (i.e. `GetTextTranslation` or `GetTextComment`)